english/TodayWasAGoodDay

```diff
@@ -0,0 +1 @@
+Today was a good day! Keep your head up, dream big and work hard to achieve it. Bless up!
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.